sites/ava-site/docs/api/chart-advisor/ChartAdvisor.zh.md

```diff
@@ -0,0 +1,118 @@
+---
+title: ChartAdvisor
+order: 1
+---
+
+`markdown:docs/common/style.md`
+
+<div class='doc-md'>
+
+```sign
+Class ChartAdvisor(config?: Partial<Pick<ChartAdvisor, 'ckbCfg' | 'ruleCfg'>> = {})
```

Comment:
Should the differences between ChartAdvisor and Advisor be explained? ChartAdvisor combines Advisor and Linter, users may not be aware of this difference.

Reply:
The difference was explained in guide/chart-advisor/ChartAdvisor. Should the API files contain such explanations? cc @neoddish